src/planner.cc

```diff
@@ -81,20 +154,42 @@ static float clamp_defining_axis_limit(const int *axes_steps,
   return new_defining_axis_limit * steps_per_mm[defining_axis];
 }
 
+// Uniformly accelerated ramp distance estimation.
+// The ramp is parametrized by v0 as the starting speed, v1 the final speed
+// and the constant acceleration.
+static double uar_distance(double v0, double v1, double acceleration) {
```

Comment:
The uar is a needlessly cryptic prefix `uar_distance()`, `uar_speed()`...
If we can use longer names we should.

Reply:
(solved)